README.md

```diff
@@ -128,6 +129,7 @@ systemsettingsrc`
       * Energy Saving
          * ./config/powermanagementprofilesrc
    * KDE Connect
+      * ./config/kde-connect
```

Comment:
On my system i dont have that folder, but ./config/kdeconnect.  Any ideas what is going on?

Reply:
You are right! I apologise, it's my mistake, no idea why I put `-` there :man_facepalming: